<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widgetMenu" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <spacer name="verticalSpacerTop">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>54</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QLabel" name="labelTitle">
         <property name="styleSheet">
          <string notr="true">font: 48pt &quot;微软雅黑&quot;;</string>
         </property>
         <property name="text">
          <string>俄罗斯方块</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <spacer name="verticalSpacerCenter">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>53</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0">
        <spacer name="horizontalSpacerLeft">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>395</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="1">
        <widget class="QWidget" name="widgetButtons" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QToolButton" name="buttonStart">
            <property name="minimumSize">
             <size>
              <width>181</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>14</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(:/assets/button.png);
font: 14pt &quot;微软雅黑&quot;;
color: rgb(0, 85, 127);</string>
            </property>
            <property name="text">
             <string>开始游戏</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="buttonHelp">
            <property name="minimumSize">
             <size>
              <width>181</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>14</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(:/assets/button.png);
font: 14pt &quot;微软雅黑&quot;;
color: rgb(0, 85, 127);</string>
            </property>
            <property name="text">
             <string>帮助</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="buttonHistory">
            <property name="minimumSize">
             <size>
              <width>181</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>14</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(:/assets/button.png);
font: 14pt &quot;微软雅黑&quot;;
color: rgb(0, 85, 127);</string>
            </property>
            <property name="text">
             <string>排行榜</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="buttonSetting">
            <property name="minimumSize">
             <size>
              <width>181</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>14</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(:/assets/button.png);
font: 14pt &quot;微软雅黑&quot;;
color: rgb(0, 85, 127);</string>
            </property>
            <property name="text">
             <string>设置</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="buttonExit">
            <property name="minimumSize">
             <size>
              <width>181</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>14</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(:/assets/button.png);
font: 14pt &quot;微软雅黑&quot;;
color: rgb(0, 85, 127);</string>
            </property>
            <property name="text">
             <string>退出</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="2">
        <spacer name="horizontalSpacerRight">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>394</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="1">
        <spacer name="verticalSpacerBottom">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>54</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="labelUser">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>32</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/assets/mask_black.png);
font: 16pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>username</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
